api/v2/checluster_types.go

```diff
@@ -661,6 +661,13 @@ type PVC struct {
 	// Storage class for the Persistent Volume Claim. When omitted or left blank, a default storage class is used.
 	// +optional
 	StorageClass string `json:"storageClass,omitempty"`
+	// StorageAccessMode are the desired access modes the volume should have.
+	// It is used to specify PersistentVolume access mode type to RWO/RWX when using per-user strategy, allowing
+	// user to re-use volume across multiple workspaces.
+	//
+	// It defaults to ReadWriteOnce if not specified
```

Comment:
The default `ReadWriteOnce` is not being set in this case,

For example, if I set:
```
    storage:
      perUserStrategyPvcConfig:
        storageAccessMode:
          - ReadWriteMany
      pvcStrategy: per-user
```

And then if I remove `perUserStrategyPvcConfig`:
```
  storage:
    pvcStrategy: per-user
```

the DWOC still has `ReadWriteMany`:
```
apiVersion: controller.devfile.io/v1alpha1
config:
  workspace:
    storageAccessMode:
      - ReadWriteMany
```

Reply:
`storageClass` and `claimSize` has the same behaviour where the defaults aren't applied:

https://github.com/eclipse-che/che-operator/blob/e2c3bf9c89240edeb8cf006721a8519c6746e395/pkg/deploy/dev-workspace-config/dev_workspace_config.go#L165

https://github.com/eclipse-che/che-operator/blob/e2c3bf9c89240edeb8cf006721a8519c6746e395/pkg/deploy/dev-workspace-config/dev_workspace_config.go#L169